src/samples/profile/v1/complete.json

```diff
@@ -0,0 +1,57 @@
+{
```

Comment:
the samples are very useful to see how the network of YAML inclusions works out to a full response